docs/getting-started/devenv/get-started-with-wp-now.md

```diff
@@ -1,3 +1,7 @@
+<div class="callout callout-info">
```

Comment:
```suggestion
<div class="callout callout-alert">
```

I feel that `callout-alert` is more suitable.

![callout](https://github.com/WordPress/gutenberg/assets/54422211/068b1d46-d897-4895-bc23-01be73f8e1ff)